web/src/stream_popover.js

```diff
@@ -76,7 +67,7 @@ export function stream_sidebar_menu_handle_keyboard(key) {
 }
 
 export function topic_sidebar_menu_handle_keyboard(key) {
```

Comment:
I believe we may no longer need this function, as the keyboard handling is now managed in popover_menus. However, it would be best to double-check before removing it.

Reply:
Removed it.